docs/source/llms/llm-evaluate/index.rst

```diff
@@ -198,10 +198,11 @@ MLflow offers a few pre-canned metrics which uses LLM as the judge. Despite the
 is the same - put these metrics in the ``extra_metrics`` argument in ``mlflow.evaluate()``. Here is the list of pre-canned
 metrics:
 
-* :py:func:`mlflow.metrics.genai.answer_similarity`: Evaluate the semantic similarity of the LLM output to the ground_truth
-* :py:func:`mlflow.metrics.genai.answer_correctness`: Evaluate the accuracy of the LLM output based on the ground_truth
-* :py:func:`mlflow.metrics.genai.answer_relevance`: Evaluate the relevance of the LLM output based on the appropriateness and applicability of the output with respect to the input.
-* :py:func:`mlflow.metrics.genai.faithfulness`: Evaluate the faithfulness of the LLM output based on how factually consistent the output is to the context.
+* :py:func:`mlflow.metrics.genai.answer_similarity`: Use this metric when you want to evaluate how similar the model generated output is compared to the information in the ground_truth. High scores mean that your model outputs contain the same information as the ground_truth, while low scores mean that outputs may disagree with the ground_truth. Note that this is a subset of answer_correctness.
```

Comment:
I think we should differentiate between the idea of "semantic similarity" and "same information" here - although since "semantic similarity" may be difficult to understand maybe something closer to "outputs mention similar things" or "outputs touch on similar topics"?

Reply:
I guess this was addressed above to be similar information.- works for me! I'll resolve this.